data/data-pipeline/data_pipeline/etl/sources/michiganejscreen/README.md

```diff
@@ -0,0 +1,11 @@
+# Michigan EJ Screen
+
+The Michigan EJ Screen description and publication can be found [here](https://deepblue.lib.umich.edu/bitstream/handle/2027.42/149105/AssessingtheStateofEnvironmentalJusticeinMichigan_344.pdf).
```

Comment:
Nit: can you standardize your references to "EJSCREEN" using one spelling? I've seen EJSCREEN, EjScreen, EJ Screen, and others. 

People use all sorts of different spellings (MiEJScreen seems to be how they refer to it) but the official EPA spelling is EJSCREEN.